### PR TITLE
add max-height and scroll to base dialog

### PR DESCRIPTION
**Pull request by mikkio-j**
Issue: #477

### CODE REVIEW

File: src/components/Dialogs/BaseDialog/BaseDialog.style.ts [hunk 1]
```diff
@@ -12,13 +12,17 @@ export const StyledContainer = styled.div`
   width: 90%;
   max-width: 440px;
   min-height: 150px;
-  overflow: hidden;
+  max-height: 550px;
+  overflow-y: scroll;
```
Comment (drillprop) [inline src/components/Dialogs/BaseDialog/BaseDialog.style.ts]: I think `overflow: auto;` should be a better choice for this. We don't want to always show scroll, we want to show the scroll only when it's needed.

File: src/components/Dialogs/BaseDialog/BaseDialog.style.ts [hunk 2]
```diff
@@ -12,13 +12,17 @@ export const StyledContainer = styled.div`
...
   margin: ${sizes(16)} auto;
   color: ${colors.white};
   background-color: ${colors.gray[700]};
   padding: var(--dialog-padding);
   box-shadow: 0 8px 8px rgba(0, 0, 0, 0.12), 0 24px 40px rgba(0, 0, 0, 0.16);
-
+  ${media.small} {
+    max-height: 650px;
+    overflow: hidden;
```
Comment (drillprop) [inline src/components/Dialogs/BaseDialog/BaseDialog.style.ts]: Not 100% sure, but with `overflow: auto` we could get rid of the `overflow: hidden` entirely, and just adjust `max-height`. Hidden overflow could be bad for laptop screens. But it needs some testing.

Also maybe we should consider setting these values based on the height of the viewport, not width.

File: src/components/SignInSteps/AccountStep.style.ts [hunk 3]
```diff
@@ -36,7 +36,7 @@ export const SubTitle = styled(Text)`
 
 export const AccountsWrapper = styled.div`
   width: 100%;
-  max-height: 300px;
+  max-height: 200px;
```
Comment (drillprop) [inline src/components/SignInSteps/AccountStep.style.ts]: ok, so we have max-height and min-height with the same value. We could just replace them with just `height`, right?

**Review by drillprop — APPROVED** [review]
Last thing 😅

**Review by drillprop — APPROVED** [review]
👍🏻

**Review by kdembler — APPROVED** [review]

**Pull request merged**